src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -157,6 +158,15 @@ private void OnTreeViewItemPreviewMouseLeftButton(object sender, MouseButtonEven
             e.Handled = !(selectedClass is RootNodeCategoryViewModel);
         }
 
+        private bool IsUserVisible(FrameworkElement element, FrameworkElement container)
```

Comment:
Well you can rest assured that *visible* means *visible to user*, so `IsElementVisible` should be good enough (and is better than `IsUserVisible`. Could you please update that? Thanks!